docs/build/tutorials/smart-contracts/verify-smart-contracts.md

```diff
@@ -8,17 +8,20 @@ If you have issues, contact us on [Discord](https://chat.avalabs.org).
 
 ## Steps
 
-Navigate to the _Code_ tab at the Explorer page for your contract's address.
+Navigate to the _Contract_ tab at the Explorer page for your contract's address.
+<img width="1359" alt="contract_tab" src="https://user-images.githubusercontent.com/73849597/144134924-e800ba23-a65d-4adb-ae4a-26f3fe631f6c.png"></img>
```

Comment:
@Julian-dev28 does Ava Labs have full control of these images? Can these images be deleted without our knowledge? Can you put these images under `avalanche-docs/static/img` directory?

Reply:
this needs to be changed to local file